### DISCUSSION TITLE
The issue of sinicization of Chinese

### DISCUSSION THREAD

**Discussion by LXW600** [meetrevision/playbook#65] [Q&A]
After installing the win11 version, after downloading the Chinese language pack according to the tutorial, the system is converted to Chinese, but after about 3 days, it becomes English and the system language cannot be changed

**Comment by HarryVasanth** [comment]
Ok this one is a bit tricky 😅

Based on the context of Win 10/11 and how the ReviOS Playbook is configured, this is a known issue caused by a default Windows background task rather than a specific bug in ReviOS itself.

### Potential reasons:

Windows language packs (Language Experience Packs and Features on Demand) heavily rely on Windows Update and background syncing to fully download, install, and verify all localised components (such as speech, handwriting, and system UI translations).

Because you applied the ReviOS Playbook and how Windows behaves, your system has the following restrictions:

1. **The Windows `Lpremove` Task:** Windows has a built-in scheduled task called `Lpremove` that runs in the background a few days after a new language is installed. If a language pack is not explicitly applied to the system's "Welcome Screen" and default system accounts, windows sometimes wrongly assumes it is "unused" and uninstalls it.
2. **ReviOS Cleanup Scripts:** The ReviOS playbook includes a `CLEANER.ps1` script that configures the Windows Disk Cleanup to automatically target and remove "Language Pack" files to save disk space (`"Language Pack" = 2`).
3. **Paused Updates:** ReviOS intentionally pauses Windows Updates until the year 2038 to prevent forced updates and bloatware. However, windows language packs (and their Local Experience Packs) rely heavily on windows update to download fully. When blocked, the language pack may install incompletely, leading to corruption or forced reversions.

### Fix

To permanently apply the Chinese language pack, you must temporarily allow Windows to download all necessary language files and set it as the default system language.

**Step 1: Unpause Windows Updates**

1. Open the **Revision Tool** (installed by the playbook, usually located at `%ProgramFiles%\Revision Tool\revitool.exe`).
2. Navigate to the **Windows Update** tweaks section and **enable/unpause** Windows Updates.

**Step 2: Clean up the Broken Language Pack**

1. Go to Windows **Settings** > **Time & language** > **Language & region**.
2. Change your Windows display language back to English temporarily.
3. Click the three dots next to the **Chinese** language pack and select **Remove**.

**Step 3: Reinstall the Language Pack**

1. In the same Settings menu, click **Add a language** and select Chinese.
2. Make sure you check all the optional language features (Basic typing, Handwriting, Speech, etc.) so that they fully download via Windows Update.
3. Check the box to **Set as my Windows display language** and click Install. Wait for the download to completely finish.

**Step 4: Copy the Language Settings to the System (Important)**
To prevent the lock screen or system services from reverting to English:

1. In the *Language & region* settings, scroll down to **Administrative language settings**.
2. Under the *Administrative* tab, click the **Copy settings** button.
3. At the bottom of the window that pops up, check the boxes for **Welcome screen and system accounts** and **New user accounts**.
4. Click **OK** and restart your computer.

**Step 5: Re-pause Updates (Optional)**
Once your computer restarts and you have verified that the entire system is properly displaying in Chinese, you can open the **Revision Tool** again and re-pause Windows Updates if you wish.
